### PR TITLE
refactor(Telephony-alias pages): Contact page alignment

### PR DESCRIPTION
**Pull request by feoche**
## Title of the Pull Requests
Contact page alignment

### Description of the Change
Contact page alignment with oui-kit

### CODE REVIEW

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 1]
```diff
@@ -32,8 +32,9 @@
                     data-ng-if="ServiceContactCtrl.isEditing"
                     data-ng-disabled="!ServiceContactCtrl.hasChanges() || ServiceContactCtrl.isUpdating || contactForm.$invalid">
                 <span data-translate="telephony_service_contact_apply"></span>
-                <spinner class="ml-1"
-                         data-ng-if="ServiceContactCtrl.isUpdating"></spinner>
+                <bouncing-box-loader data-size="s"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: +`d-inline-block align-middle`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 2]
```diff
@@ -49,19 +50,20 @@
...
+            <select id="contactLegalForm"
+                    data-ng-if="ServiceContactCtrl.isEditing"
+                    name="contactLegalForm"
+                    class="form-control"
+                    data-ng-model="ServiceContactCtrl.directoryForm.legalForm"
+                    data-ng-change="ServiceContactCtrl.onChangeLegalForm()"
+                    data-ng-options="legalForm as ('telephony_service_contact_legalform_' + legalForm | translate) for legalForm in ['individual', 'professional', 'corporation']"
+                    required>
+            </select>
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`
ref: https://getbootstrap.com/docs/3.3/css/#forms-controls-static

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 3]
```diff
@@ -81,14 +81,17 @@
                    data-ng-model="ServiceContactCtrl.directoryForm.name"
                    autofocus
                    required />
-            <div data-ng-messages="contactForm.contactLastName.$error"
-                 data-ng-if="contactForm.contactLastName.$invalid"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 4]
```diff
@@ -98,23 +101,24 @@
...
             <input type="text"
                    id="contactFirstName"
                    name="contactFirstName"
                    class="form-control"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.firstName"
                    required />
-            <div data-ng-messages="contactForm.contactFirstName.$error"
-                 data-ng-if="contactForm.contactFirstName.$invalid"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 5]
```diff
@@ -123,14 +127,15 @@
...
                    data-translate="telephony_service_contact_occupation"></label>
-            <p data-ng-if="!ServiceContactCtrl.isEditing"
-               data-ng-bind="ServiceContactCtrl.directoryForm.occupation || '-'"></p>
             <input type="text"
                    id="contactOccupation"
                    name="contactOccupation"
                    class="form-control"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.occupation" />
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 6]
```diff
@@ -152,18 +155,20 @@
                    data-ng-model-options="{ debounce: 500 }"
                    data-ng-pattern="/^\d{9,}$/"
                    autofocus />
-            <div data-ng-messages="contactForm.contactSiret.$error"
-                 data-ng-if="contactForm.contactSiret.$invalid && contactForm.contactSiret.$dirty"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 7]
```diff
@@ -183,14 +186,17 @@
                    data-ng-pattern="/^\d{4}\w$/"
                    data-ng-model="ServiceContactCtrl.directoryForm.ape"
                    data-ng-model-options="{ debounce: 500 }" />
-            <div data-ng-messages="contactForm.contactApe.$error"
-                 data-ng-if="contactForm.contactApe.$invalid && contactForm.contactApe.$dirty"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 8]
```diff
@@ -210,14 +214,17 @@
                    data-ng-model="ServiceContactCtrl.directoryForm.socialNomination"
                    data-ng-touched
                    data-required />
-            <div data-ng-messages="contactForm.contactSocialNomination.$error"
-                 data-ng-if="contactForm.contactSocialNomination.$invalid"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 9]
```diff
@@ -226,14 +233,15 @@
...
                    data-translate="telephony_service_contact_social_nomination_extra"></label>
-            <p data-ng-if="!ServiceContactCtrl.isEditing"
-               data-ng-bind="ServiceContactCtrl.directoryForm.socialNominationExtra || '-'"></p>
             <input type="text"
                    id="contactSocialNominationExtra"
                    name="contactSocialNominationExtra"
                    class="form-control"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.socialNominationExtra" />
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 10]
```diff
@@ -242,14 +250,15 @@
...
                    data-translate="telephony_service_contact_email"></label>
-            <p data-ng-if="!ServiceContactCtrl.isEditing"
-               data-ng-bind="ServiceContactCtrl.directoryForm.email || '-'"></p>
             <input type="email"
                    id="contactEmail"
                    name="contactEmail"
                    class="form-control"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.email" />
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 11]
```diff
@@ -263,16 +272,16 @@
...
-               data-ng-if="!ServiceContactCtrl.isEditing"
-               data-ng-bind="ServiceContactCtrl.directoryForm.country || '-'"></p>
             <input type="text"
                    id="contactCountry"
                    name="contactCountry"
                    class="form-control text-uppercase"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.country"
                    disabled />
+            <p class="form-control-plaintext text-uppercase"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 12]
```diff
@@ -292,14 +299,17 @@
                    data-ng-model="ServiceContactCtrl.directoryForm.postCode"
                    data-ng-model-options="{ debounce: 500 }"
                    required />
-            <div data-ng-messages="contactForm.contactPostCode.$error"
-                 data-ng-if="contactForm.contactPostCode.$invalid"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 13]
```diff
@@ -308,47 +318,50 @@
             <label for="contactCity"
                    class="control-label">
                 <span data-translate="telephony_service_contact_city"></span>
-                <spinner data-ng-if="ServiceContactCtrl.isCityListLoading"></spinner>
+                <bouncing-box-loader data-size="s"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: +`d-inline-block align-middle ml-X`?

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 14]
```diff
@@ -308,47 +318,50 @@
...
-                </div>
-                -->
+            <input type="text"
+                   data-ng-if="ServiceContactCtrl.isEditing"
+                   id="contactCity"
+                   name="contactCity"
+                   class="form-control"
+                   data-ng-model="ServiceContactCtrl.directoryForm.city"
+                   required />
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 15]
```diff
@@ -308,47 +318,50 @@
...
+                  data-ng-if="contactForm.contactCity.$touched"
+                  class="help-block"
+                  role="alert">
+                 <span data-ng-message="required"
+                       data-translate="telephony_service_contact_must_fill">
+                 </span>
+            </span>
+
+            <!--
+            <select id="contactCity"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: @frenautvh @Jisay Can be removed?

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 16]
```diff
@@ -358,74 +371,77 @@
...
             <input type="text"
                    id="contactUrbanDistrict"
                    name="contactUrbanDistrict"
                    class="form-control"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.urbanDistrict"
                    required />
-            <div data-ng-messages="contactForm.contactUrbanDistrict.$error"
-                 data-ng-if="contactForm.contactUrbanDistrict.$invalid"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 17]
```diff
@@ -358,74 +371,77 @@
...
                    data-translate="telephony_service_contact_cedex"></label>
-            <p data-ng-if="!ServiceContactCtrl.isEditing"
-               data-ng-bind="ServiceContactCtrl.directoryForm.cedex || '-'"></p>
             <input type="text"
                    id="contactCedex"
                    name="contactCedex"
                    class="form-control"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.cedex" />
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 18]
```diff
@@ -358,74 +371,77 @@
...
                    data-translate="telephony_service_contact_waynumber"></label>
-            <p data-ng-if="!ServiceContactCtrl.isEditing"
-               data-ng-bind="ServiceContactCtrl.directoryForm.wayNumber || '-'"></p>
             <input type="text"
                    id="contactWaynumber"
                    name="contactWaynumber"
                    class="form-control"
                    data-ng-if="ServiceContactCtrl.isEditing"
                    data-ng-model="ServiceContactCtrl.directoryForm.wayNumber" />
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 19]
```diff
@@ -358,74 +371,77 @@
...
+                    name="contactWaynumberExtra"
+                    class="form-control text-uppercase"
+                    data-ng-model="ServiceContactCtrl.directoryForm.wayNumberExtra">
+                <option data-ng-bind="" value=""></option>
+                <option data-ng-repeat="key in ServiceContactCtrl.wayNumberExtraEnum track by $index"
+                        data-ng-bind="key"
+                        value="{{ key }}">
+                </option>
+            </select>
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 20]
```diff
@@ -434,10 +450,9 @@
             <label for="contactWayname"
                    class="control-label">
                 <span data-translate="telephony_service_contact_wayname"></span>
-                <spinner data-ng-if="ServiceContactCtrl.isWayListLoading"></spinner>
+                <bouncing-box-loader data-size="s"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: +`d-inline-block align-middle ml-X`?

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 21]
```diff
@@ -446,14 +461,17 @@
                    data-ng-model="ServiceContactCtrl.directoryForm.wayName"
                    data-ng-model-options="{ debounce: 300 }"
                    required />
-            <div data-ng-messages="contactForm.contactWayname.$error"
-                 data-ng-if="contactForm.contactWayname.$invalid"
+            <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 22]
```diff
@@ -474,56 +492,58 @@
...
                 </label>
-                <p data-ng-if="!ServiceContactCtrl.isEditing"
-                   data-ng-bind="ServiceContactCtrl.directoryForm.PJSocialNomination || '-'"></p>
                 <input type="text"
                        id="contactPjdenomination"
                        name="contactPjdenomination"
                        class="form-control"
                        data-ng-if="ServiceContactCtrl.isEditing"
                        data-ng-model="ServiceContactCtrl.directoryForm.socialNomination" />
+                <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 23]
```diff
@@ -474,56 +492,58 @@
...
-                <p class="text-capitalize"
+                <select id="directoryServiceCode"
+                        data-ng-if="ServiceContactCtrl.isEditing"
+                        name="directoryServiceCode"
+                        class="form-control text-capitalize"
+                        data-ng-model="ServiceContactCtrl.directoryForm.directoryServiceCode"
+                        data-ng-disabled="!ServiceContactCtrl.directoryCodes"
+                        data-ng-options="opt.directoryServiceCode as opt.directoryServiceDescription for opt in ServiceContactCtrl.directoryCodes">
+                </select>
+                <p class="form-control-plaintext text-capitalize"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 24]
```diff
@@ -474,56 +492,58 @@
...
                        data-translate="telephony_service_contact_universal_directory_publish"></label>
-                <p class="text-capitalize"
+                <select id="displayUniversalDirectory"
+                        data-ng-if="ServiceContactCtrl.isEditing"
+                        name="displayUniversalDirectory"
+                        class="form-control text-capitalize"
+                        data-ng-model="ServiceContactCtrl.directoryForm.displayUniversalDirectory"
+                        data-ng-options="opt as (opt ? 'yes' : 'no' | translate) for opt in [true, false]">
+                </select>
+                <p class="form-control-plaintext text-capitalize"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 25]
```diff
@@ -533,35 +553,33 @@
...
                            data-translate="telephony_service_contact_display_marketing"></label>
-                    <p class="text-capitalize"
+                    <select id="displayMarketingDirectory"
+                            data-ng-if="ServiceContactCtrl.isEditing"
+                            name="displayMarketingDirectory"
+                            class="form-control text-capitalize"
+                            data-ng-model="ServiceContactCtrl.directoryForm.displayMarketingDirectory"
+                            data-ng-options="opt as (opt ? 'yes' : 'no' | translate) for opt in [true, false]">
+                    </select>
+                    <p class="form-control-plaintext text-capitalize"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 26]
```diff
@@ -533,35 +553,33 @@
...
                            data-translate="telephony_service_contact_search_reverse"></label>
-                    <p class="text-capitalize"
+                    <select id="displaySearchReverse"
+                            data-ng-if="ServiceContactCtrl.isEditing"
+                            name="displaySearchReverse"
+                            class="form-control text-capitalize"
+                            data-ng-model="ServiceContactCtrl.directoryForm.displaySearchReverse"
+                            data-ng-options="opt as (opt ? 'yes' : 'no' | translate) for opt in [true, false]">
+                    </select>
+                    <p class="form-control-plaintext text-capitalize"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 27]
```diff
@@ -570,17 +588,16 @@
...
                            data-translate="telephony_service_contact_display_name"></label>
-                    <p class="text-capitalize"
+                    <select id="contactDisplayFirstName"
+                            data-ng-if="ServiceContactCtrl.isEditing"
+                            name="contactDisplayFirstName"
+                            class="form-control text-capitalize"
+                            data-ng-model="ServiceContactCtrl.directoryForm.displayFirstName"
+                            data-ng-options="opt as ServiceContactCtrl.getDisplayFirstNameOptions(opt) for opt in [true, false]">
+                    </select>
+                    <p class="form-control-plaintext text-capitalize"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 28]
```diff
@@ -589,17 +606,15 @@
...
                            class="control-label"
                            data-translate="telephony_service_contact_address_publish"></label>
-                    <p class="text-capitalize"
+                    <select id="displayOnlyCity"
+                            class="form-control text-capitalize"
+                            data-ng-if="ServiceContactCtrl.isEditing"
+                            data-ng-model="ServiceContactCtrl.directoryForm.displayOnlyCity"
+                            data-ng-options="opt as ('telephony_service_contact_address_publish_' + (opt ? 'city' : 'full') | translate) for opt in [false, true]">
+                    </select>
+                    <p class="form-control-plaintext text-capitalize"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 29]
```diff
@@ -611,14 +626,15 @@
...
                     </label>
-                    <p data-ng-if="!ServiceContactCtrl.isEditing"
-                       data-ng-bind="ServiceContactCtrl.directoryForm.lineDescription || '-'"></p>
                     <input type="text"
                            id="contactServiceDescription"
                            name="contactServiceDescription"
                            class="form-control"
                            data-ng-if="ServiceContactCtrl.isEditing"
                            data-ng-model="ServiceContactCtrl.directoryForm.lineDescription" />
+                    <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 30]
```diff
@@ -627,7 +643,8 @@
             <div class="form-group">
                 <label class="control-label"
                        data-translate="telephony_service_contact_status"></label>
-                <p data-ng-bind="'telephony_service_contact_sync' + (ServiceContactCtrl.directoryForm.status === 'todo' ? '_pending' : '') | translate">
+                <p class="form-control-plaintext"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `form-control-plaintext` => `form-control-static`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 31]
```diff
@@ -649,8 +666,9 @@
                         data-ng-if="ServiceContactCtrl.isEditing"
                         data-ng-disabled="!ServiceContactCtrl.hasChanges() || ServiceContactCtrl.isUpdating || contactForm.$invalid">
                     <span data-translate="telephony_service_contact_apply"></span>
-                    <spinner class="ml-1"
-                             data-ng-if="ServiceContactCtrl.isUpdating"></spinner>
+                    <bouncing-box-loader data-size="s"
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: Loader must be before your translation.
+`d-inline-block align-middle mr-2`

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html (hunk 15 above)
Comment (feoche) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: I hope so ¯\_(ツ)_/¯

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html (hunk 1 above)
Comment (antleblanc) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: `bouncing-box-loader` + `span` (order)

**Comment by antleblanc** [comment]
@feoche could you please resolve conflict :candy:

**Pull request merged**